app/src/main/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -2,4 +2,5 @@
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
     <background android:drawable="@color/ic_launcher_background"/>
     <foreground android:drawable="@mipmap/ic_launcher_foreground"/>
-</adaptive-icon>
\ No newline at end of file
+    <monochrome android:drawable="@mipmap/ic_launcher_foreground"/>
```

Comment:
https://github.com/rkkr/simple-keyboard/blob/master/app/src/main/res/mipmap-xxhdpi/ic_launcher_foreground.png is not monochrome, so it should just screw up the color overlay.